xml/admin_rbd.xml

```diff
@@ -84,27 +84,40 @@
   <sect2 xml:id="ceph-rbd-cmds-create-ec">
    <title>Creating a Block Device Image in an Erasure Coded Pool</title>
    <para>
-    As of &productname; 5, it is possible to store data of a block device image
+    It is possible to store data of a block device image
     directly in erasure coded (EC) pools. A &rbd; image consists of
     <emphasis>data</emphasis> and <emphasis>metadata</emphasis> parts. You can
-    store only the 'data' part of a &rbd; image in an EC pool. The pool needs
-    to have the 'overwrite' flag set to <emphasis>true</emphasis>, and that is
+    store only the data part of a &rbd; image in an EC pool. The pool needs
+    to have the <option>overwrite</option> flag set to <emphasis>true</emphasis>, and that is
     only possible if all OSDs where the pool is stored use &bluestore;.
    </para>
    <para>
-    You cannot store the image's 'metadata' part in an EC pool. You need to
+    You cannot store the image's metadata part in an EC pool. You can
     specify the replicated pool for storing the image's metadata with the
     <option>--pool=</option> option of the <command>rbd create</command>
-    command.
+    command or specify <option>pool/</option> as a prefix to the image name.
    </para>
    <para>
-    Use the following steps to create an RBD image in a newly created EC pool:
+    Create an EC pool:
    </para>
-<screen>&prompt.cephuser;<command>ceph</command> osd pool create <replaceable>POOL_NAME</replaceable> 12 12 erasure
-&prompt.cephuser;<command>ceph</command> osd pool set <replaceable>POOL_NAME</replaceable> allow_ec_overwrites true
-
-#Metadata will reside in pool "<replaceable>OTHER_POOL</replaceable>", and data in pool "<replaceable>POOL_NAME</replaceable>"
-&prompt.cephuser;<command>rbd</command> create <replaceable>IMAGE_NAME</replaceable> --size=1G --data-pool <replaceable>POOL_NAME</replaceable> --pool=<replaceable>OTHER_POOL</replaceable></screen>
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>POOL_NAME</replaceable> 12 12 erasure
+&prompt.cephuser;ceph osd pool set <replaceable>POOL_NAME</replaceable> allow_ec_overwrites true</screen>
+   <para>
+     Specify the replicated pool for storing metadata:
+   </para>
+<screen>
+&prompt.cephuser;rbd create <replaceable>IMAGE_NAME</replaceable> --size=1G --data-pool <replaceable>EC_POOL</replaceable> --pool=<replaceable>POOL</replaceable>
+</screen>
+   <para>
+     Or:
+   </para>
+<screen>
+&prompt.cephuser;rbd create <replaceable>POOL/IMAGE_NAME</replaceable> --size=1G --data-pool EC_POOL
+</screen>
+   <para>
+     Use the following command to list the image of the metadata pool:
```

Comment:
Hm. I am not sure I like it. What I meant that when you run `rbd list $pool` or `rbd list --pool $pool` it will show images created with metadata pool `$pool`.

Actually, after your changes above, we probably don't need this paragraph at all. It looks clear enough.

Reply:
Will remove it :)